plugins/plugin-ssh-key/che-plugin-ssh-key-server/src/main/java/org/eclipse/che/plugin/ssh/key/HttpSshServiceClient.java

```diff
@@ -84,8 +81,7 @@ public void createPair(SshPairDto sshPair) throws ServerException {
     @Override
     public SshPairDto getPair(String service, String name) throws ServerException, NotFoundException {
         try {
-            final String url = UriBuilder.fromUri(sshUrl)
-                                         .path(SshService.class, "getPair")
+            final String url = UriBuilder.fromUri(sshUrl + "/getPair")
```

Comment:
Fixed API urls, moved DTO generation stuff in `pom.xml` from `che-core-api-ssh` to `che-core-api-ssh-shared`, Add test that checks API urls with the help of `UriBuilder` and `SshService.class`.